pmm-app/src/pmm-qan/panel/components/Details/Details.messages.ts

```diff
@@ -2,7 +2,7 @@ export const Messages = {
   noExamplesFound: 'Sorry, no examples found for this query',
   noClassicExplain: 'No classic explain found',
   noJsonExplain: 'No JSON explain found',
-  cantExtractTables: 'Couldn&apos;t get tables info neither from example nor explain',
+  cantExtractTables: 'Couldn`t get tables info neither from example nor explain',
```

Comment:
Actually, that character should not replace the usual apostrophe `'`. This looks like a case where double quotes are appropriate. ESLint shouldn't be complaining.

Reply:
Also, I suggest we ask @PaulJacobs-percona to check the message.

Reply:
Shorter, same meaning (to my ear):
"No table info from example nor explain"

Reply:
@PaulJacobs-percona thanks, fixed